File: examples/functions/python/auto-infer-no-requirements/handler.py
import json


def _inferred_imports_marker():
    # FastFN should infer and persist these packages into requirements.txt.
    import requests  # noqa: F401
    return "requests"


def handler(event):
    return {
        "status": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json.dumps(
            {
                "runtime": "python",
                "function": "auto-infer-no-requirements",
                "inference": "enabled",
            },
            separators=(",", ":"),
        ),
    }

File: examples/functions/node/auto-infer-create-package/handler.js
module.exports.handler = async () => {
  // FastFN should infer this dependency and create package.json automatically.
  if (false) {
    require("uuid");
  }

  return {
    status: 200,
    headers: { "Content-Type": "application/json" },
    body: JSON.stringify({
      runtime: "node",
      function: "auto-infer-create-package",
      inference: "enabled",
    }),
  };
};
